bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsManager.java

```diff
@@ -233,6 +235,13 @@ public List<File> getFullFilledLedgerDirs() {
         return filledDirs;
     }
 
+    /**
+     * @return warn threshold ledger dirs.
+     */
+    public List<File> getWarnLedgerDirs() {
```

Comment:
What's the difference with `diskWritable` of the newly introduced interface?

Reply:
`warnLedgerDirs` is only used for trigger under warn threshold event. has no effects on writing and reading.